.github/workflows/backend-tests.yml

```diff
@@ -6,15 +6,67 @@ on:
 
   workflow_dispatch:
 
+env:
+  BACKEND_DEBUG: 0
+  POSTGRES_PASSWORD: example
+  POSTGRES_USER: example
+  DB_HOST: db
+  DB_NAME: postgres
+  FRONTEND_TARGET: production
+  NODE_IMAGE: lts-alpine
+  JWT_SECRET_KEY: example
+  DISABLE_AUTHENTICATION: true
+
+
 jobs:
   build:
     runs-on: ubuntu-latest
+    timeout-minutes: 20
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
```

Comment:
At this time, it seems that the `lts-alpine` version is 16 (see [this page](https://hub.docker.com/_/node)).